frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -786,6 +850,9 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
                   title={`${status.phase}: ${status.message}`}
                 />
               )}
+              {customResource && (
+                <AlertToCreateCustomResource cr={customResource} obj={props.obj} />
```

Comment:
```suggestion
                <InitializationResourceAlert initializationResource={initializationResource} obj={props.obj} />
```

Reply:
If `obj` is always a CSV, it would be better to call the prop `csv`. Otherwise it's confusing because there are two resources (the CSV and the initialization resource). `obj` is too vague.